doc/developers-guide/accumulators.md

```diff
@@ -103,7 +117,7 @@ The notion of an exit point is a bit vague, because sending a stanza out of Mong
 A single stanza entering the system creates an acc, but then may result in multiple stanzas going out - for example changing a privacy list triggers presence updates and a roster push.
 It is in fact hard to determine when the process has really been finished.
 The approach is to pass along one acc and, when something is just about to be sent out, either send a text and return the acc or pass on a clone of the acc and return the original one.
-This gives us an option to record a fact that a stanza has been sent out.
+This gives us an option to record a fact that a stanza has been sent out: every sending function calls `mongoose_acc:record_sending`, which gives you an opportunity to consult `send_result` key to check what send attempts have been made and with what effect.
```

Comment:
This gives us an option to record a fact that a stanza has been sent out.
Every sending function calls `mongoose_acc:record_sending`, which gives you an opportunity to consult the `send_result` key to check what send attempts have been made and with what effect.